### PR TITLE
Fix ARG005 per file ignore

### PR DESCRIPTION
**Pull request by MaximSmolskiy**
### Describe your change:

Fix `ARG005` per file ignore

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: machine_learning/linear_discriminant_analysis.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def valid_input(
     input_type: Callable[[object], num],  # Usually float or int
     input_msg: str,
     err_msg: str,
-    condition: Callable[[num], bool] = lambda x: True,
+    condition: Callable[[num], bool] = lambda _: True,
```
Comment (imSanko) [inline machine_learning/linear_discriminant_analysis.py:259]: Why do we need to change the preset parameter's?

File: pyproject.toml [hunk 2]
```diff
@@ -86,7 +86,6 @@ max-complexity = 17  # default: 10
 "graphs/minimum_spanning_tree_prims.py" = ["SIM114"]
 "hashes/enigma_machine.py" = ["BLE001"]
 "machine_learning/decision_tree.py" = ["SIM114"]
-"machine_learning/linear_discriminant_analysis.py" = ["ARG005"]
 "machine_learning/sequential_minimum_optimization.py" = ["SIM115"]
```
Comment (imSanko) [inline pyproject.toml:89]: Guess we shouldn't as to change the code !!

**Review by imSanko — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (cclauss) [reply pyproject.toml:89]: Why do you "guess" that?

Comment (imSanko) [reply pyproject.toml:89]: Is that simply great to remove a line?

**Review by imSanko — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (imSanko) [reply pyproject.toml:89]: Is that great while removing a line, while making it a fix !!

**Review by cclauss — APPROVED** [review]
Yes.  Removing the line is enabling a test (adding more tests, not modifying or removing tests) so it seems quite valid to me.

**Pull request merged**